src/components/implementation/fork_tests/malloc_fork/malloc_fork.c

```diff
@@ -0,0 +1,30 @@
+#include <cos_component.h>
+#include <print.h>
+#include <quarantine.h>
+#include <test_malloc_comp.h>
+
+#define ITER (1024)
+u64_t meas[ITER];
```

Comment:
Is this dead code?  Looks like it.  The comment following this also seems out of sync with the rest of the component.